dashboard/src/components/Status.tsx

```diff
@@ -0,0 +1,47 @@
+/*/**
+ * @file Status.tsx
+ * The Status component:
+ * This component is used to display the status of a deployment.
+ * It uses the tailwindcss classes to implement the status.
+ * There are 3 possible statuses:
+ * 1. Deployed
+ * 2. Superseded
+ * 3. Failed
+ * The status is indicated by a green orb, a neutral grey orb , or a red orb.
+ *
+ * @param statusCode: enum StatusCode
+ *
+ *
+ */
+import React from "react";
+import "../index.css";
+export default function Status({
+  statusCode,
+  isRefreshable = false
+}: {
+  statusCode: "Deployed" | "Superseded" | "Failed";
+  isRefreshable: boolean;
+}): JSX.Element {
+  const statusVariants = {
```

Comment:
place content declarations on top of file  (after imports)

Reply:
This is declared in global.tsx now